[2022-01-02 12:43:10,999] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2021-12-28T09:00:00+00:00 [queued]>
[2022-01-02 12:43:11,013] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2021-12-28T09:00:00+00:00 [queued]>
[2022-01-02 12:43:11,013] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-01-02 12:43:11,013] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2022-01-02 12:43:11,013] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-01-02 12:43:11,021] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2021-12-28T09:00:00+00:00
[2022-01-02 12:43:11,024] {standard_task_runner.py:54} INFO - Started process 16184 to run task
[2022-01-02 12:43:11,089] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2021-12-28T09:00:00+00:00', '--job_id', '40', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/var/folders/yv/xtbkkbn565j6hhnrh5chcn4r0000gn/T/tmpo2nf4vg1']
[2022-01-02 12:43:11,091] {standard_task_runner.py:78} INFO - Job 40: Subtask twitter_aluraonline
[2022-01-02 12:43:16,140] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2021-12-28T09:00:00+00:00 [running]> on host MacBook-Pro-de-Roberto.local
[2022-01-02 12:43:21,271] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2022-01-02 12:43:21,274] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-28T09:00:00.00Z&end_time=2021-12-29T09:00:00.00Z
[2022-01-02 12:43:23,419] {logging_mixin.py:112} WARNING - /Users/roberto-almeida/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-01-02 12:43:23,742] {logging_mixin.py:112} INFO - /Users/roberto-almeida/datapipeline/datalake/bronze/twitter_aluraonline/extract_date=2021-12-28/AluraOnline_20211228.json
[2022-01-02 12:43:23,743] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-28T09:00:00.00Z&end_time=2021-12-29T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe17gd9e8if23z9l5e13v3amjum4d
[2022-01-02 12:43:23,744] {logging_mixin.py:112} WARNING - /Users/roberto-almeida/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-01-02 12:43:24,032] {logging_mixin.py:112} INFO - /Users/roberto-almeida/datapipeline/datalake/bronze/twitter_aluraonline/extract_date=2021-12-28/AluraOnline_20211228.json
[2022-01-02 12:43:24,033] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-28T09:00:00.00Z&end_time=2021-12-29T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe17gd7au28mcdex9wtvvb87445fh
[2022-01-02 12:43:24,035] {logging_mixin.py:112} WARNING - /Users/roberto-almeida/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-01-02 12:43:24,323] {logging_mixin.py:112} INFO - /Users/roberto-almeida/datapipeline/datalake/bronze/twitter_aluraonline/extract_date=2021-12-28/AluraOnline_20211228.json
[2022-01-02 12:43:24,324] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-28T09:00:00.00Z&end_time=2021-12-29T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe17gd6zpj5vy5h2zag1ru1eoopkt
[2022-01-02 12:43:24,325] {logging_mixin.py:112} WARNING - /Users/roberto-almeida/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-01-02 12:43:24,626] {logging_mixin.py:112} INFO - /Users/roberto-almeida/datapipeline/datalake/bronze/twitter_aluraonline/extract_date=2021-12-28/AluraOnline_20211228.json
[2022-01-02 12:43:24,627] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-28T09:00:00.00Z&end_time=2021-12-29T09:00:00.00Z&next_token=b26v89c19zqg8o3fpe17gb2vfn0u8a7r4s6cfnnjrcfwd
[2022-01-02 12:43:24,628] {logging_mixin.py:112} WARNING - /Users/roberto-almeida/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2022-01-02 12:43:24,895] {logging_mixin.py:112} INFO - /Users/roberto-almeida/datapipeline/datalake/bronze/twitter_aluraonline/extract_date=2021-12-28/AluraOnline_20211228.json
[2022-01-02 12:43:24,902] {taskinstance.py:1070} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20211228T090000, start_date=20220102T154311, end_date=20220102T154324
[2022-01-02 12:43:26,109] {local_task_job.py:102} INFO - Task exited with return code 0
